### ISSUE TITLE
Payload Validation

### ISSUE THREAD

**Issue by timomeh** [timomeh/WBA2SS15AfshariPoplawskiMaemecke#14]

**Comment by Plsr** [comment]
That would've be done for every Route I guess, making this a task for every one of us?

**Comment by timomeh** [comment]
Yeah, [express-validator](https://github.com/ctavan/express-validator) is a great module for this. But we would need to talk together about this and time's running low.
